frontend/packages/dev-console/integration-tests/features/addFlow/create-from-devfile.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Create Application from git form
+    As a user, I want to deploy an application from git repo with devfile
+
+
+    Background:
+        Given user is at developer perspective
+        And user is at Add page
+        And user has created namespace starts with "aut-addflow-devfile"
+
+
+    @smoke
+    Scenario: From Devfile card on +Add page
+        Given user is at add page
+        Then user will see From Devfile card on the page
+
+
+    @smoke
+    Scenario: Deploy new application from git repo with devfile from Add page
+        Given user is at add page
+        When user clicks on From Devfile card
+        And user enters Git Repo url as "https://github.com/maysunfaisal/node-bulletin-board"
+        And user enters Name as "node-bulletin-board"
+        And user clicks on Create button
+        Then user will be redirected to Topology page
+        And user is able to see workload "node-bulletin-board" in topology page
+
+
+    @regression
+    Scenario: Deploy new application from git repo with devfile from topology page
```

Comment:
Won't there be an edit scenario option?

Reply:
I have added the edit scenario but it's incomplete. There is a bug raised and we are unaware of the feature. Once the bug is resolved, we will update the scenario. We reached to this conclusion after discussing it with Andrew.

Reply:
Do have a task assigned in your name for  it as and when the bug is fixed you can include the changes in the script

Reply:
Not yet. I will create one when the bug will get fixed.